### ISSUE TITLE
[BUG] - Can't upgrade NextUI (package lost)

### ISSUE THREAD

**Issue by Cierra-Runis** [nextui-org/nextui#4234]
### NextUI Version

2.4.8

### Describe the bug

```bash
nextui upgrade

NextUI CLI v0.4.0

√ Select the components to upgrade » @nextui-org/react@2.4.8 -> 2.6.0
╭────────────────────── Component ───────────────────────╮
│  @nextui-org/react              2.4.8  ->  2.6.0       │
╰────────────────────────────────────────────────────────╯

1 minor

√ Would you like to proceed with the upgrade? » Yes

yarn add @nextui-org/react@2.6.0
➤ YN0000: · Yarn 4.2.2
➤ YN0000: ┌ Resolution step
➤ YN0082: │ @nextui-org/snippet@npm:2.2.0: No candidates found
➤ YN0000: └ Completed in 1s 337ms
➤ YN0000: · Failed with errors in 1s 353ms

Unexpected error. Please report it as a bug:
Error: Command failed: yarn add @nextui-org/react@2.6.0
    at checkExecSyncError (node:child_process:890:11)
    at execSync (node:child_process:962:15)
    at file:///C:/Users/28642/AppData/Roaming/npm/node_modules/nextui-cli/dist/index.js:37:259
    at new Promise (<anonymous>)
    at he (file:///C:/Users/28642/AppData/Roaming/npm/node_modules/nextui-cli/dist/index.js:37:137)
    at Command.zn (file:///C:/Users/28642/AppData/Roaming/npm/node_modules/nextui-cli/dist/index.js:80:2544)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Command.parseAsync (C:\Users\28642\AppData\Roaming\npm\node_modules\nextui-cli\node_modules\commander\lib\command.js:935:5) {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 24004,
  stdout: null,
  stderr: null
}
```

Checking by `npm view @nextui-org/snippet versions`, `2.6.0` doesn't exist.

Or using canary version:

```bash
PS D:\Desktop\NEW\Modified Desktop\note-of-me.top> yarn add @nextui-org/react@canary
➤ YN0000: · Yarn 4.2.2
➤ YN0000: ┌ Resolution step
➤ YN0082: │ @nextui-org/alert@npm:0.0.0-canary-20241205201328: No candidates found
➤ YN0000: └ Completed in 0s 557ms
➤ YN0000: · Failed with errors in 0s 570ms
```

### Steps to Reproduce the Bug or Issue

```bash
nextui upgrade
```

Or:

```bash
yarn add @nextui-org/react@canary
```

### Expected behavior

Upgrade successfully.

### Operating System Version

Windows

### Browser

Edge

**Comment by Cierra-Runis** [comment]
It's version 2.6.2 now, but I am facing the issue what [here](https://github.com/nextui-org/nextui/issues/2735#issuecomment-2059385978) announced...

However, by using NextUI CLI and selecting all components it has, I fix these issues.

**Comment by wingkwong** [comment]
@Cierra-Runis do you mean the original issue has been fixed in 2.6.2?

**Comment by Cierra-Runis** [comment]
> @Cierra-Runis do you mean the original issue has been fixed in 2.6.2?

No, sorry. I fixed my issues by NextUI CLI and I didn't try 2.6.2.

**Comment by wingkwong** [comment]
Got it.
